packages/plugins/contracts/foundry/Sphinx.sol

```diff
@@ -257,8 +264,6 @@ abstract contract Sphinx {
         sphinxModifierEnabled = false;
     }
 
-    function run() public virtual;
```

Comment:
This line forced the user to include `override` in their `run` function signature